osquery/tables/applications/browser_firefox.cpp

```diff
@@ -26,13 +26,13 @@ namespace tables {
 
 namespace {
 
-/// Each home directory will include custom extensions.
+/// Each home directory will include custom extensions. Comma separated list of paths.
 #if defined(__APPLE__)
-#define kFirefoxPath "/Library/Application Support/Firefox/Profiles/"
+#define kFirefoxPaths "/Library/Application Support/Firefox/Profiles/"
 #elif defined(__linux__)
-#define kFirefoxPath "/.mozilla/firefox/"
+#define kFirefoxPaths "/.mozilla/firefox/,/snap/firefox/common/.mozilla/firefox/"
```

Comment:
Why not define this as an array?

Reply:
I kept the code similar to the chrome one.

Reply:
I guess that makes sense.. But if we're just going to split it later, it feels better to hard code the array.

Reply:
ok then, please have a look at it. Thanks!